domain/src/main/java/com/teamagam/gimelgimel/domain/dynamicLayers/DynamicLayerVisibilityChange.java

```diff
@@ -0,0 +1,18 @@
+package com.teamagam.gimelgimel.domain.dynamicLayers;
+
+import com.teamagam.gimelgimel.domain.base.visiblity.VisibilityChange;
+
+public class DynamicLayerVisibilityChange extends VisibilityChange {
+
+  private final String mId;
```

Comment:
maybe ren to `mDynamicLayerId`, I thought for a moment the id belongs to the `change` itself